test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -66,6 +66,21 @@ SMTCheckerTest::SMTCheckerTest(string const& _filename): SyntaxTest(_filename, E
 		m_ignoreCex = true;
 	else
 		BOOST_THROW_EXCEPTION(runtime_error("Invalid SMT counterexample choice."));
+
+	auto const& os = m_reader.stringSetting("SMTIgnoreOS", "none");
+	for (auto&& o: os | ranges::views::split(',') | ranges::to<vector<string>>())
```

Comment:
It's a bit confusing to call the whole string `os`.
```suggestion
	string const& ignoreOSSetting = m_reader.stringSetting("SMTIgnoreOS", "none");
	for (string const& os: ignoreOSSetting | ranges::views::split(',') | ranges::to<vector<string>>())
```